kubernetes/deployment.yml

```diff
@@ -15,3 +16,14 @@ spec:
         image: gcr.io/google_containers/kube-state-metrics:v0.4.1
         ports:
         - containerPort: 8080
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 8
+          timeoutSeconds: 4
+        resources:
+          limits:
+            cpu: .1
+            memory: 75M
```

Comment:
While CPU resource seems fine, as `kube-state-metrics` quite literally just lists/watches, I'm wondering how you came up this as an upper bound for memory, as this highly depends on number of objects in your cluster. I'm fine with giving a suggestion of memory limits, just wondering what kind of cluster size/object count we're talking about here.